practice-one/src/styles/components/introduction.css

```diff
@@ -0,0 +1,48 @@
+.introduction-container {
+  display: flex;
+  justify-content: space-between;
+  gap: 300px;
+}
+
+.introduction-content-container {
+  padding-top: 50px;
+}
+
+.people-trust-icon {
+  display: flex;
+  text-align: center;
+  box-shadow: drop;
```

Comment:
Recheck this I dont think this is properly value

Reply:
i fixed it